packages/xo-server/src/api/pci.mjs

```diff
@@ -0,0 +1,18 @@
+import { defer } from 'golike-defer'
+
+export const set = defer(async function ($defer, { pci, hide }) {
+  const xapi = this.getXapi(pci)
+
+  if (hide !== undefined) {
+    await xapi.call(`PCI.${hide ? 'hide' : 'unhide'}`, pci._xapiRef)
+    $defer.onFailure(() => xapi.call(`PCI.${!hide ? 'hide' : 'unhide'}`, pci._xapiRef))
```

Comment:
this is triggered in case of reboot failure. Should we also try to reboot he host again after ?

Reply:
Host reboot is required to apply the change.
If we undo the change, we do not need to reboot the host